http-clients/java-nio-client/src/test/java/software/amazon/awssdk/http/nio/java/internal/S3JavaHttpClientIntegratedTest.java

```diff
@@ -36,95 +56,108 @@ private void createS3AsyncClient(SdkAsyncHttpClient myClient) {
-        createJavaHttpClientHTTP1();
+    public void Es3PutLargeObjectWithJavaHttpClientHTTP1Test() {
+        createAsyncClientWithJavaHttpClientHTTP1();
         putObjectTest(s3Async, LARGEKEY, LARGESTRING);
     }
 
     @Test
-    public void s3PutLargeObjectWithJavaHttpClientHTTP2Test() {
-        createJavaHttpClientHTTP2();
+    public void Fs3PutLargeObjectWithJavaHttpClientHTTP2Test() {
+        createAsyncClientWithJavaHttpClientHTTP2();
         putObjectTest(s3Async, LARGEKEY, LARGESTRING);
     }
 
     @Test
-    public void s3GetObjectWithJavaHttpClientHTTP1Test() {
-        createJavaHttpClientHTTP1();
+    public void Gs3GetObjectWithJavaHttpClientHTTP1Test() {
+        createAsyncClientWithJavaHttpClientHTTP1();
         getObjectTest(s3Async);
     }
 
     @Test
-    public void s3GetObjectWithJavaHttpClientHTTP2Test() {
-        createJavaHttpClientHTTP2();
+    public void Hs3GetObjectWithJavaHttpClientHTTP2Test() {
+        createAsyncClientWithJavaHttpClientHTTP2();
         getObjectTest(s3Async);
     }
 
     @Test
-    public void s3DeleteObjectWithJavaHttpClientHTTP1Test() {
-        createJavaHttpClientHTTP1();
+    public void Is3GetObjectWithSyncHttpClient() {
+        createS3Client();
+        ResponseBytes responseBytes = s3Client.getObject(GetObjectRequest.builder().bucket(BUCKET).key(KEY).build(),
+                ResponseTransformer.toBytes());
+        assertEquals(responseBytes.asUtf8String(), VALUE);
+    }
+
+    @Test
+    public void Js3DeleteObjectWithJavaHttpClientHTTP1Test() {
+        createAsyncClientWithJavaHttpClientHTTP1();
         deleteObjectTest(s3Async);
     }
 
 
     private static void createBucketTest(S3AsyncClient s3Async) {
-        s3Async.createBucket(CreateBucketRequest.builder().bucket(BUCKET).build()).join();
+        try (s3Async) {
+            s3Async.createBucket(CreateBucketRequest.builder().bucket(BUCKET).build()).join();
+        } catch (Exception e) {
+            e.printStackTrace();
+        }
     }
 
     private static void putObjectTest(S3AsyncClient s3Async, String key, String value) {
-        CompletableFuture<PutObjectResponse> future = s3Async.putObject(
-                PutObjectRequest.builder()
-                        .bucket(BUCKET)
-                        .key(key)
-                        .build(),
-                AsyncRequestBody.fromString(value)
-        );
-        future.whenComplete((resp, err) -> {
-            try (s3Async) {
-                if (resp != null) {
-                    System.out.println("my response: " + resp);
-                } else {
+        try (s3Async) {
+            CompletableFuture<PutObjectResponse> future = s3Async.putObject(
+                    PutObjectRequest.builder()
+                            .bucket(BUCKET)
+                            .key(key)
+                            .build(),
+                    AsyncRequestBody.fromString(value)
+            );
+            future.whenComplete((resp, err) -> {
```

Comment:
Just remove this complete. `join()` will throw if there's an error and the test will fail which is what we want.